okio/src/jvmTest/kotlin/okio/BufferedSinkTest.kt

```diff
@@ -27,27 +28,23 @@ import okio.TestUtil.segmentSizes
 import org.junit.Assert.assertEquals
 import org.junit.Assert.fail
 import org.junit.Test
-import org.junit.runner.RunWith
-import org.junit.runners.Parameterized
-import org.junit.runners.Parameterized.Parameters
 
-@RunWith(Parameterized::class)
+@Burst
 class BufferedSinkTest(
   factory: Factory,
 ) {
-  interface Factory {
-    fun create(data: Buffer): BufferedSink
+  enum class Factory {
+    BUFFER {
+      override fun create(data: Buffer) = data
+      override fun toString() = "Buffer"
```

Comment:
We can rename all these in a follow-up to have these nicer names like you did for the new one at the bottom. No need for screaming case anymore.

Reply:
On it!